setup.py

```diff
@@ -4,39 +4,22 @@
     long_description = fh.read()
 
 dependencies = [
-    "chia-blockchain==1.8.1",
-    "packaging==23.0",
+    "chia-blockchain==2.0.0",
 ]
 
 dev_dependencies = [
-    "build",
-    "click~=8.1.3",
-    "coverage",
-    "pre-commit",
-    "pylint",
-    "pytest",
-    "pytest-asyncio>=0.18.1",  # require attribute 'fixture'
-    "pytest-monitor; sys_platform == 'linux'",
-    "pytest-xdist",
-    "twine",
-    "isort",
-    "faker",
-    "flake8",
-    "mypy",
-    "black==22.10.0",
-    "aiohttp_cors",  # For blackd
-    "ipython",  # For asyncio debugging
-    "pyinstaller==5.0",
-    "pytest",
-    "pytest-asyncio",
-    "pytest-env",
-    "types-aiofiles",
-    "types-click",
-    "types-cryptography",
-    "types-pkg_resources",
-    "types-pyyaml",
-    "types-setuptools",
-    "types-docutils",
+    "pre-commit==3.3.3",
```

Comment:
Do we both have dependabot enabled here and also keep up with the PRs it generates to update versions?

Reply:
I don't think we have dependabot on this repo, but it does seem like a useful thing to do

Reply:
updated to include dependabot config